internal/worker/main.go

```diff
@@ -166,6 +171,22 @@ func (w *worker) processUser(user slackGo.User) (*data.User, error) {
 	}, nil
 }
 
+func (w *worker) processConversations() error {
+	w.logger.Info("getting conversations from api")
+	conversations, err := w.getConversations()
+	if err != nil {
+		return errors.Wrap(err, "failed to get conversations from Slack API")
+	}
+
+	for _, conversation := range conversations {
+		if err := w.processor.UpsertConversation(&conversation); err != nil {
```

Comment:
You can update your `.Upsert` method for Conversations interface to take a batch and perform batch upserting, if you had such method, you could just made `.Upsert(...conversations)`. If you need any example, just touch me in TG